libphidgets/CMakeLists.txt

```diff
@@ -1,11 +1,18 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(libphidgets)
 
-find_package(catkin REQUIRED COMPONENTS rosbuild roslib mk rosbuild rospack)
+find_package(catkin REQUIRED COMPONENTS mk rospack)
```

Comment:
is `mk` a catkin package?
or does it need to be `find_package`d separately?
Do we need to declare `CATKIN_DEPENDS` or `DEPENDS` to them in catkin_package()?

Do we need to `find_package` libusb-dev?